storage/tests/unit/test_bucket.py

```diff
@@ -208,12 +208,12 @@ def test_ctor_explicit(self):
         now = datetime.datetime.utcnow().replace(tzinfo=pytz.UTC)
 
         config = self._make_one(
-            bucket, bucket_policy_only_enabled=True, bucket_policy_only_locked_time=now
+            bucket, uniform_bucket_level_access_enabled=True, uniform_bucket_level_access_locked_time=now
         )
 
         self.assertIs(config.bucket, bucket)
-        self.assertTrue(config.bucket_policy_only_enabled)
-        self.assertEqual(config.bucket_policy_only_locked_time, now)
+        self.assertTrue(config.uniform_bucket_level_access_enabled)
+        self.assertEqual(config.uniform_bucket_level_access_locked_time, now)
```

Comment:
Likewise, we need to continue to assert values for the deprecated properties here.

We also need separate testcases for passing `bucket_policy_only_enabled` / `bucket_policy_only_locked_time` to the constructor.